### PR TITLE
feat(settings): add internal url to jellyfin settings form

### LINKED ISSUE #194: Change Jellyfin IP
### Description

When the IP address or hostname of your jellyfin server changes you have to update it in the settings.json file. There is no clear indicator of this in the settings and even worse is if your account is a jellyfin account the error just states incorrect username or password

### Desired Behavior

A settings options under the jellyfin tab for `local jellyfin address` which could work the same as the external URL option (If my understanding of that option is correct and it also just updates the json)

Also a better error message along the lines of `Cannot connect to jellyfin
server` when trying to log in.

### Additional Context

_No response_

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

### PR DESCRIPTION
**Pull request by jeaboswell**
#### Description

Add an input field to the Jellyfin settings to allow users to change the internal URL of the server without editing the configuration file directly.

#### Screenshot (if UI-related)

New Settings View

#### To-Dos

- [x] Successful build `yarn build`
- [x] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Fixes #194

### CODE REVIEW

File: src/components/Settings/SettingsJellyfin.tsx [hunk 1]
```diff
@@ -89,6 +90,10 @@
       /^(?:(?:(?:https?):)?\/\/)(?:\S+(?::\S*)?@)?(?:(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*\.?)(?::\d{2,5})?(?:[/?#]\S*)?$/,
       intl.formatMessage(messages.validationUrl)
     ),
+    jellyfinInternalUrl: Yup.string().matches(
+      /^(?:(?:(?:https?):)?\/\/)(?:\S+(?::\S*)?@)?(?:(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*\.?)(?::\d{2,5})?(?:[/?#]\S*)?$/,
```
[the remark this answers is not in the document]
Comment (jeaboswell) [reply src/components/Settings/SettingsJellyfin.tsx]: @Fallenbagel Any idea why it would complain about this one, but not the existing external one?

**Comment by jeaboswell** [comment]
I can't help but notice that when I run `yarn i18n:extract`, there are a lot more changes than just what I have done.  Are the locale files still being maintained?

**Comment by Fallenbagel** [comment]
> I can't help but notice that when I run `yarn i18n:extract`, there are a lot more changes than just what I have done.  Are the locale files still being maintained?

The reason is, some of the ui changes like "jellyseerr" are simply done on locale instead of the component files for better compatibility with merging from upstream

File: src/components/Settings/SettingsJellyfin.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (fallenbagel) [reply src/components/Settings/SettingsJellyfin.tsx]: This regex was what we arrived at back when we implemented external hostname, due to baseurls and trailing slashes and stuff. There's probably a better way to handle what we do there. Something like how radarr/sonarr baseurls are handled in the services with a seperate field 🤔

For now I guess it's fine to ignore this

[the remark this answers is not in the document]
Comment (fallenbagel) [reply src/components/Settings/SettingsJellyfin.tsx]: @jeaboswell how about something like this? What do you think?
```regex
/^(https?:\/\/)?(?:[\w-]+\.)*[\w-]+(?::\d{2,5})?(?:\/[\w-]+)*(?:\/)?$/gm
```

**Review by Fallenbagel — APPROVED** [review]

**Pull request merged**